pkg/gardenlet/controller/controllerinstallation/controllerinstallation/reconciler.go

```diff
@@ -469,3 +487,35 @@ func (r *Reconciler) reconcileGardenAccessSecret(ctx context.Context, controller
 
 	return accessSecret, accessSecret.Reconcile(ctx, r.SeedClientSet.Client())
 }
+
+// BootstrapControlPlaneNodeFunc adapts host network, replicas, tolerations and usable ports range for autonomous shoot
+// clusters if necessary.
+func (r *Reconciler) BootstrapControlPlaneNodeFunc(obj runtime.Object) error {
```

Comment:
Should this have a better name to reflect what it's really doing? 
```suggestion
func (r *Reconciler) MutateSpecForControlPlaneNodeBootstrapping(obj runtime.Object) error {
```

not sure if this is _much_ better, but maybe a _bit_?